src/Route/AdminPoolLoader.php

```diff
@@ -49,12 +48,12 @@ public function __construct(Pool $pool, array $adminServiceIds, ContainerInterfa
         $this->container = $container;
     }
 
-    public function supports($resource, ?string $type = null)
+    public function supports($resource, string $type = null): bool
```

Comment:
You'll get a phpcs issue here (and same for other changes).

Even if `?` is not required, a phpcs rule is requiring it when `null` is used as a default value.

Reply:
I saw this, what should I do?
If I add the `?` the Interface is incompatible, else the phpcs fails.

Reply:
I would be really surprised if adding the `?` is incompatible with the Interface.

`?string $type = null` and `string $type = null` is the same thing.